RecoParticleFlow/PFClusterTools/src/PFEnergyCalibration.cc

```diff
@@ -299,7 +314,39 @@ PFEnergyCalibration::bEtaBarrel(double x) const {
 
   } else { 
 
-    return fbEtaBarrel->Eval(x); 
+    return fbEtaBarrelEH->Eval(x); 
+
+  }
+}
+
+double 
+PFEnergyCalibration::aEtaBarrelH(double x) const { 
+
+  if ( pfCalibrations ) { 
+
+    BinningPointByMap point;
+    point.insert(BinningVariables::JetEt, x);
+    return pfCalibrations->getResult(PerformanceResult::PFfaEta_BARREL,point); 
```

Comment:
here and in a most of other places ```PFfaEta_BARREL``` (the old key) is used.
Is this going to be changed to ```PFfaEta_BARRELH``` once the calibrations are available?